doc/Developing.md

```diff
@@ -470,6 +487,16 @@ If you already have Xcode installed, but you see the `xcode-select: error: tool
 
 The solution is the same if you have updated to `10.14` (Mojave) and you can see the `gyp: No Xcode or CLT version detected!` error. More details [here](https://github.com/nodejs/node-gyp#on-macos).
 
+### RHEL7
+
+See prerequisites for [`node-keytar`](#prerequisites)
+
+You might need to set the following environment variable before doing `yarn`:
```

Comment:
It might be useful to describe the problem this addresses. Something like `At runtime, you may encounter an error with the message <error message from issue>. This can be fixed by forcing pre-built dependencies to rebuild from source by setting the following environment variable before `yarn`:` (you may also need to clean, since that script gets run on `install`, so if you try setting the environment variable when you already have the package, it may not get run.)